documentation/docs/client/setup/logging.md

```diff
@@ -30,16 +38,16 @@ val client = ChatClient.Builder("apiKey", context)
      .build()
 ```
 
-## Finding logs
+## Filtering logs
 
-All SDK log tags have the `Chat:` prefix, so you can filter for that those in the logs:
+All SDK log tags `Chat:` as a prefix that you can use when filtering logs.
 
 ```bash
 adb logcat com.your.package | grep "Chat:"
 ```
 
 Here's a set of useful tags for debugging network communication:
 
-- `Chat:Http`
-- `Chat:Events`
-- `Chat:SocketService`
+- `Chat:Http` to see HTTP requests made from the `ChatClient` and the responses returned by Stream.
+- `Chat:Events` to see a list of [events](https://getstream.github.io/stream-chat-android/stream-chat-android-client/stream-chat-android-client/io.getstream.chat.android.client.events/-chat-event/index.html) that the `ChatClient` emits.
```

Comment:
We'll want to update this link when things are moved to Docusaurus, could be worth adding a TODO for somehow (or omitting for now)

Reply:
My plan is to keep a list of these old links and use a regex to replace them with analogous pages when we migrate. Do you know if the Dokka docs will have the same paths after the base url for the github pages site? (IE: of `https://getstream.github.io/stream-chat-android/stream-chat-android-client/stream-chat-android-client/io.getstream.chat.android.client.events/-chat-event/index.html` will everything after `https://getstream.github.io/stream-chat-android` remain the same or are we changing the paths for the API reference?)

Reply:
I have already create a task for it

Reply:
Dokka Reference will live into our Docusaurus project, and it should have similar url from the root of this project